ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml

```diff
@@ -24,16 +24,28 @@ releases:
       version: "4.15"
 resources:
   '*':
-    limits:
-      memory: 4Gi
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: unit
-  commands: echo "test"
-  container:
-    from: src
+- always_run: false
+  as: e2e-cluster-density-v2-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "3"
```

Comment:
Just thinking outloud here, could we get away w/ a SNO or reducing the replica count to 1 for the compute nodes?

Reply:
Yes. Modified it to to "1"